### ISSUE TITLE
Double comma in line 24 of cpsr_validate_input.py

### ISSUE THREAD

**Issue by ridwan-shaikh** [sigven/cpsr#18]
Hi Sigven,

Just to let you know, there is a double comma on line 24 after choice=[0,1] of cpsr_validate_input.py

```
2019-10-11 15:16:40 - cpsr-validate-input - INFO - STEP 0: Validate input data
  File "/home/.conda/envs/pcgr/bin/cpsr_validate_input.py", line 24
    parser.add_argument('diagnostic_grade_only',type=int, default=0,choices=[0,1],,help="Green virtual panels only (Genomics England PanelApp)")
                                                                                  ^
SyntaxError: invalid syntax

```

**Comment by sigven** [comment]
Dear @ridwan-shaikh,
Thanks a lot for the notification. A stupid mistake. Will make an updated fixed release later today :-).
best,
Sigve

**Comment by sigven** [comment]
Fixed now (v0.5.1)
